shard.yml

```
@@ -29,7 +29,7 @@ dependencies:
   # Models
   placeos-models:
     github: placeos/models
-    version: ~> 5.0
+    version: ~> 5.5.1
```

```suggestion
    version: ~> 5.5
```

Pins like `~> 5.5.1` only encapsulate versions up to and not including `5.6`. Better to pin to `~> 5.5`